pyenvbuilder/main.py

```diff
@@ -4,12 +4,14 @@
 import argparse
 import logging
 from pyenvbuilder import __version__
+from .commands import check
 
 
 logger = logging.getLogger(__name__)
+COMMANDS = [check]
 
 
-def launch(log_level="INFO"):
+def launch(command=None, files=[], log_level="INFO"):
```

Comment:
command can't be `None`. See suggestion above... we can talk about it in more details.